src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index];
```

Comment:
A few points about this part:
1) Please pay attention to this part in the common mistakes document: https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level
Because now you create a Random instance every time you call getRandomColor(), this is insufficient.
2) You call Color.values() twice, these parts should be extracted to a separate variable.